packages/analytics/amplify_analytics_pinpoint/pubspec.yaml

```diff
@@ -35,3 +35,12 @@ dev_dependencies:
   amplify_lints: ^2.0.0
   flutter_test:
     sdk: flutter
+  pigeon: ^4.2.5
+
+# The following section is specific to Flutter.
```

Comment:
You can probably remove the comment. We have this comment in a few other files. I think it is part of the auto generated comments from `flutter create` and was never removed from those files, but probably should be.